circle.yml

```diff
@@ -28,3 +32,6 @@ workflows:
       - linux
       - linux-release
       - linux-performance
+notify:
+  webhooks:
+    - url: https://6a4b8e77.ngrok.io/circle
```

Comment:
We could also wait to merge this until the bot is hosted on Vapor Cloud

Reply:
Yeah, let's wait until we can get this hosted somewhere else. I can work on this today.